FLEDGE.md

```diff
@@ -121,6 +122,23 @@ The `adComponents` field contains the various ad components (or "products") that
 The browser will provide protection against microtargeting, by only rendering an ad if the same rendering URL is being shown to a sufficiently large number of people (e.g. at least 100 people would have seen the ad, if it were allowed to show).  As discussed in the [Outcome-Based TURTLEDOVE](https://github.com/WICG/turtledove/blob/master/OUTCOME_BASED.md) proposal, this threshold applies only to the rendered creative; all of the metadata associated with ad bidding is not under any such restriction.  Since a single interest group can carry multiple possible ads that it might show, the group will have an opportunity to re-bid another one of its ads to act as a "fallback ad" any time its most-preferred choice is below threshold.  This means that a small, specialized interest group that is still below the daily-update threshold could still choose to participate in auctions, bidding with a more-generic ad until the group becomes large enough.
 
 
+#### 1.3 Permission Delegation
+
+When one domain calls joinAdInterestGroup() or leaveAdInterestGroup() for an interest group with a different owner, the browser will fetch the URL https://owner.domain/.well-known/interest-group/permissions/domain.of.frame.that.called.the.method, without credentials, using the Network Partition Key of the frame that invoked the method. The fetched response should have a JSON MIME type and be of the format:
```

Comment:
> 1. can we clarify `owner.domain` to be the interest group owner domain?

Done (and shortened + described the other domain)

> 2. I was imagining one file listing the permitted domains, rather than one file per permitted/disallowed domain. Hmm, is there prior art of folks using .well-known files for one or the other?

I'm not sure - there are a bunch of .well-known paths, none commonly used. Origin Policy (which I think is now defunct) used a versioning system for subdirectories and redirects between them.  Looks like .well-known/core allows (optional) query search params on paths.  I've never heard of that .well-known URL, though, and it's not clear if by "prior art" you mean "widely deployed and supported" or "registered and specified".

I'm concerned about a list potentially getting large, and having to load it into browser-process memory.  With one URL each, we can safely load it into browser process memory, and have a reasonable response body cap.  Also, generally we only care about one entry, so having a long list that we don't care about seems not useful.  With a list, we'll need a limit, and people may at some point run into if - if a DSP lets all publishers it has contracts with add IGs (or runs code in the context of publisher pages to add a user to IGs, and is comfortable with that security model), the list could get very large.

> 3. is "Network Partition Key" a well known term?  is it in the Fetch spec?  if so, can link to it's definition here?  I'm just worried this may not be a common term for the FLEDGE explainer's audience.

It's in the fetch spec.  Added a link.